conf/default-breakpoints.yaml

```diff
@@ -1,18 +1,23 @@
 breakpoints:
   desktop-wide:
+    max-width: 100vw
     min-width: 1440px
     content: 1440px
     col: xl
+    default: true
```

Comment:
what about `{{!mobileFirst}}` and then doing `{{mobileFirst}}` on mobile breakpoint?